dev-python/toposort/toposort-1.6.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Implements a topological sort algorithm"
+HOMEPAGE="https://gitlab.com/ericvsmith/toposort"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
```

Comment:
IUSE and RESTRICT are already set by `distutils_enable_tests` if they are required, so we do not need to set these explicitly.

Reply:
Removed explicit IUSE="test" and RESTRICT